geoportailv3/mymaps.py

```diff
@@ -0,0 +1,321 @@
+        if self.title is not None:
+            return self.title.replace("'", "_")
+        return None
+
+    def todict(self):
+        def convert_datetime(value):
+            if value is not None:
+                return value.strftime("%Y-%m-%d %H:%M:%S")
+            else:
+                return None
+
+        for c in self.__table__.columns:
+            if isinstance(c.type, DateTime):
+                value = convert_datetime(getattr(self, c.name))
+            else:
+                value = getattr(self, c.name)
+            yield(c.name, value)
+
+    def __iter__(self):
+        """Returns an iterable that supports .next()
+            so we can do dict(sa_instance)
+        """
+        return self.todict()
+
+    @staticmethod
+    def get(id):
+        """ Get map by its id. """
+        return DBSession.query(Map).get(id)
+
+    @staticmethod
+    def belonging_to(user):
+        """ Get maps that belong to user. """
+        maps = DBSession.query(Map).filter(Map.user_login == user)\
+            .order_by("category_id asc,title asc").all()
+
+        return [{'title': map.title,
+                 'uuid': map.uuid,
+                 'public': map.public,
+                 'category': map.category.name
+                 if map.category_id is not None else None} for map in maps]
+
+
+class Feature(Base):
+    __tablename__ = 'feature'
+    __template__ = 'tooltips/category.mako'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(Unicode(50))
+    description = Column(Unicode)
+    image = Column(Unicode(255))
+    thumbnail = Column(Unicode(255))
+    color = Column(Unicode(255))
+    stroke = Column(Integer, default=2)
+    is_label = Column(Boolean, default=False)
+    linestyle = Column(Integer, default=0)
+    show_orientation = Column(Boolean, default=False)
+    geometry = Column(Geometry(srid=2169))
+    map_id = Column(Unicode, ForeignKey('map.uuid'))
+    symbol_id = Column(Integer)
+    size = Column(Float, default=40)
+    angle = Column(Float, default=0)
+    font_size = Column(Integer, default=15)
+
+    def __init__(self, feature=None):
+        if feature:
+            self.__update__(feature)
+
+    def __update__(self, feature):
+        self.name = feature.properties.get('name')
+        self.description = feature.properties.get('description')
+        self.thumbnail = feature.properties.get('thumbnail')
+        self.image = feature.properties.get('image')
+        self.color = feature.properties.get('color')
+        self.stroke = feature.properties.get('stroke')
+        self.is_label = feature.properties.get('isLabel')
+        self.show_orientation = feature.properties.get('showOrientation')
+        self.linestyle = feature.properties.get('linestyle')
+
+        if (feature.properties.get('size') is not None and
+                unicode(feature.properties.get('size')).isnumeric()):
+            self.size = feature.properties.get('size')
+        else:
+            self.size = 40
```

Comment:
Here you repeat `feature.properties.get('size')` three times. How about the following:

```py
        size = feature.properties.get('size')
        if size is not None and unicode(size).isnumeric():
            self.size = size
        else:
            self.size = 40
```

or

```py
        size = feature.properties.get('size')
        self.size = size if size is not None and unicode(size).isnumeric() else 40
```

?